spoon-bytecode/pom.xml

```diff
@@ -0,0 +1,545 @@
+    <artifactId>spoon-bytecode</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0-SNAPSHOT</version>
+    <name>Spoon Bytecode</name>
+    <description>Extension of spoon for bytecode through decompilation</description>
+    <url>http://spoon.gforge.inria.fr/</url>
+    <inceptionYear>2007</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>CeCILL-C</name>
+            <comments>French equivalent to LGPL</comments>
+            <url>http://www.cecill.info/licences/Licence_CeCILL-C_V1-en.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <organization>
+        <name>Inria</name>
+        <url>http://www.inria.fr</url>
+    </organization>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/INRIA/spoon/issues</url>
+    </issueManagement>
+
+    <scm>
+        <url>https://github.com/INRIA/spoon</url>
+        <connection>scm:git:https://github.com/INRIA/spoon.git</connection>
+        <developerConnection>scm:git:ssh://github.com/INRIA/spoon.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <properties>
+        <java.src.version>1.8</java.src.version>
+        <java.test.version>1.8</java.test.version>
+        <runtime.log>target/velocity.log</runtime.log>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <version.cfr>0.132.0</version.cfr>
+    </properties>
+
+    <distributionManagement>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+        <snapshotRepository>
+            <id>maven.inria.fr-snapshot</id>
+            <name>Maven Repository for Spoon Snapshots</name>
+            <url>http://maven.inria.fr/artifactory/spoon-public-snapshot</url>
+        </snapshotRepository>
+        <site>
+            <id>gforge.inria.fr-site</id>
+            <name>inria</name>
+            <url>scp://scm.gforge.inria.fr/home/groups/spoon/htdocs/mvnsites/spoon-core</url>
+        </site>
+    </distributionManagement>
+
+    <repositories>
+        <!-- This repository is actually needed for revapi to compare against the last Spoon Snapshot -->
+        <repository>
+            <id>maven.inria.fr-snapshot</id>
+            <name>Maven Repository for Spoon Snapshots</name>
+            <url>http://maven.inria.fr/artifactory/spoon-public-snapshot</url>
+        </repository>
+        <!-- This repository is needed for cfr -->
+        <repository>
+            <id>inria</id>
+            <name>triskell-public-release</name>
+            <url>http://maven.inria.fr/artifactory/triskell-public-release</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>fr.inria.gforge.spoon</groupId>
+            <artifactId>spoon-core</artifactId>
+            <version>7.2.0-SNAPSHOT</version>
```

Comment:
It would be really great/easier for our Maven config to have a spoon-pom, which would be our parent pom and in which we would define the different plugin configurations. Moreover it would allow us to release more easily.

Reply:
Yes I agree. I admit this was a bit lazy ;-)

Reply:
Does this means, that I should change the parent `<packaging>jar</packaging>` to  `<packaging>pom</packaging>` ? Do I need to create a sub module just for the spoon-core artifact?

Reply:
> Does this means, that I should change the parent <packaging>jar</packaging> to <packaging>pom</packaging> ?

yep

> Do I need to create a sub module just for the spoon-core artifact?

yep

Actually you might want to create a folder `spoon-pom` for this module and then don't forget the `<relativePath>` info on both spoon-core pom and spoon-bytecode

Reply:
Ok I got. Thanks!